examples/data_view_field_editor_example/kibana.jsonc

```diff
@@ -1,16 +1,12 @@
 {
   "type": "plugin",
   "id": "@kbn/data-view-field-editor-example-plugin",
-  "owner": "@elastic/kibana-app-services",
+  "owner": "@elastic/kibana-data-discovery",
```

Comment:
Looks like this was still assigned to app services, but I'm assuming it should be us now.